### PR TITLE
Feature/world view

### PR DESCRIPTION
**Pull request by Fayth**
Implemented world view map mode. Details: http://forum.vcmi.eu/viewtopic.php?p=13865#13865

### CODE REVIEW

**Comment by Fayth** [comment]
The last update should fix all important remaining problems (incorrect display of rotated tiles/roads/rivers on some scales + flags drawing).

File: client/mapHandler.cpp [hunk 1]
```diff
@@ -1077,6 +1350,9 @@ CMapHandler::CMapHandler()
 	frameW = frameH = 0;
 	graphics->FoWfullHide = nullptr;
 	graphics->FoWpartialHide = nullptr;
+	normalBlitter = new CMapNormalBlitter(this);
+	worldViewBlitter = new CMapWorldViewBlitter(this);
+	puzzleViewBlitter = new CMapPuzzleViewBlitter(this);
```
Comment (alexvins) [inline client/mapHandler.cpp]: Blitters desctrution missing?

Comment (Fayth) [inline client/mapHandler.cpp]: Yea, good catch. Thanks.

**Comment by alexvins** [comment]
No more known issues remaining. @DjWarmonger, are any objections to merge?

**Comment by DjWarmonger** [comment]
It's all up to you :)

**Comment by DjWarmonger** [comment]
Since this is new and significant feature, please update Changelog as well.

**Pull request merged**